generators/app/templates/src/app/screens/RecoverPassword/layout.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import i18next from 'i18next';
+
+import FormInput from '~components/FormInput';
+import PATHS from '~components/Routes/paths';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+  onEmailChange: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+}
+
+function RecoverPassword({ onSubmit, onEmailChange }: Props) {
+  return (
+    <form className={`column center full-width m-top-8 ${styles.formContainer}`} onSubmit={onSubmit}>
+      <div className="column center m-bottom-3">
+        <h1 className="m-bottom-1">{i18next.t('RecoverPassword:recoverPassword')}</h1>
+        <h2>{i18next.t('RecoverPassword:recoverPasswordDescription')}</h2>
+      </div>
+      <div className={`column m-bottom-2 ${styles.sectionContainer}`}>
+        <FormInput
```

Comment:
Is this outer div needed? It only have one child and the form component is already a flex column. Just to check.